public/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
```

Comment:
This file is autogenerated, please delete.

Reply:
Done